examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,94 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
+
+# Form the rhs of the dynamics equations
+
+RHS = mm_full.inv()*f_full
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+states = (x1, x2, u1, u2)
+
+# Form the kinetic energy to attach to one of the EOM instances as an output
+# equation
+
+KE = 0.5 * (m1*u1**2 + m2*u2**2)
+out_eqns = {"kinetic_energy": KE}
```

Comment:
`Since everything is a variable in the equations of motion, this should map a symbol to an expression instead of mapping a string to an expression. That seems to be more consistent.`

Are you meaning something along the lines of

```python
KE = symbols('KE')
out_eqns = {KE: expr}
```

`Also, should we allow simply an iterable of expressions too? That's the way most things are passed into the *Methods classes, for example.`

Are you meaning

```python
[1]: out_eqns = {'Kinetic Energy': [expr1, expr2, ...]}
```

or 

```python
[2]: out_eqns = [expr1, expr2, ...]
```

The reason I chose to use a string is so each equation could be differentiated by a distinct label. This would be useful when there are a variety of types of output equations being used as each one would be able to be labeled distinctly its name of the equation (and could later be pulled by name).